COMPONENTS.markdown

```diff
@@ -57,6 +60,40 @@ service exposes to client code.
 - Proxy Attestation Server: this is the attestation service proper, which
   can be contacted by clients to authenticate an isolate enrolled in the service.
 
+## Native modules
+In addition to WebAssembly, Veracruz can execute native code, which has the
+benefit of increasing performance (native [VOD](https://github.com/veracruz-project/video-object-detection)
+executes ~35% faster than WebAssembly) and simplifying the build of
+complex software packages, at the cost of reducing portability.
+Native code execution is achieved by the Native Module Manager described above.
+Here is a summary of the several ways to execute native code in Veracruz:
+
+- Static native module:
+  + Part of the runtime
```

Comment:
The sub-bullets here are difficult to read. It will be better to write a paragraph for each bullet.

Reply:
Replaced the bullet points with a table

Reply:
I replaced the bullet points with a table